ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -40,70 +40,77 @@ This ACP does not introduce these, but some form of asynchronous execution is re
 
-As there is one chunk per second, the gas limit is simply the gas capacity added per second, $R$, of ACPs 103 and 176.
+The block builder is expected to include transactions based on the most recently settled state and to apply worst-case bounds on the execution of the ancestor blocks prior to the most recently settled block.
 
-A chunk that is filled under criterion (2) may have leftover capacity due to imperfect packing and MUST _donate_ this residual capacity to the next chunk to allow for full utilisation of the chain's gas throughput.
- This is not to say that execution of the queue should be paused until the next second, but that the transaction's allocated chunk may well be in the future.
-For clarification, see the [alternative framing of chunk filling](#alternative-framing-of-chunk-filling).
+The worst-case bounds enforce minimum balances of sender accounts and the maximum required base fee. The worst-case bounds are described [below](#block-validity-and-building).
+
+Prior to adding a proposed block to consensus, all validators MUST verify that the block builder correctly enforced the worst-case bounds while building the block. This guarantees that the block can be executed successfully if it is accepted.
 
 > [!NOTE]
-> A side effect of chunk donation and when it is invoked is that transactions using more than a single chunk of gas MAY induce empty _interim_ chunks.
+> The worst-case bounds guarantee does not provide assurance about whether or not a transaction will revert nor whether its computation will run out of gas by reaching the specified limit. The verification only ensures the transaction is capable of paying for the accrued fees.
 
-> [!WARNING]
-> A chunk filled due to queue exhaustion MUST NOT donate remaining capacity as this would result in excess load since the next known transaction won't be available for immediate execution.
+#### Accepting blocks
 
-A completed chunk is comprised of (a) the updated state root; and (b) the set of receipts for transactions executed in the chunk.
-In addition to interim chunks, a chunk MAY also be empty if the queue was exhausted by the previous chunk and no block was agreed on by the next second.
-An empty chunk, by definition, has an empty set of receipts and the same state root as its predecessor.
+Once a block is marked as accepted by consensus, the block is enqueued in a FIFO execution queue.
 
-#### Recording state
+#### Executing blocks
 
-A block with timestamp $t_B$ MUST record the post-execution state root with a delay of exactly $d$ seconds, i.e. that which was determined by the (possibly empty) chunk at time $t_C = t_B - d$.
-Similarly, it MUST record the transaction-receipt root for all transactions executed as part of chunks in the half-open range $t_C \in (t_{B-1} - d, t_B - d]$.
-It is at this point that a transaction is said to be _settled_ on the blockchain.
+There is a constantly alive block executor whose job is to execute blocks taken off the FIFO execution queue.
```

Comment:
```suggestion
Each live avalanchego client runs a block executor that executes the blocks from the FIFO execution queue.
```
is the block executor a sub-service of avalanchego that every node runs? Or is it a centralized service whose results are pushed to all nodes in the network? unclear

Reply:
The latter. I've updated your suggestion a little, removing redundant words and `avalanchego`.